web-app-router/scenes/Portal/Profile/DangerZone/page/index.tsx

```diff
@@ -0,0 +1,48 @@
+"use client";
+
+import { SizingWrapper } from "@/components/SizingWrapper";
+import { UserInfo } from "../../layout/UserInfo";
+import { TYPOGRAPHY, Typography } from "@/components/Typography";
+import { DecoratedButton } from "@/components/DecoratedButton";
+import { Dialog } from "@/components/Dialog";
+import { DialogOverlay } from "@/components/DialogOverlay";
+import { DialogPanel } from "@/components/DialogPanel";
+import { useState } from "react";
+import { DeleteAccountDialog } from "../DeleteAccountDialog";
+
+export const DangerZone = () => {
+  const [open, setOpen] = useState<boolean>(false);
+
+  return (
+    <SizingWrapper className="py-8">
+      <div className="grid gap-y-8">
+        <UserInfo color="pink" name="Lisa" email="lisa@toolsforhumanity.org" />
+
```

Comment:
Why is this hard coded?

Reply:
If we need to apply logic already, then we need to merge #524 first